packages/shared/src/utils/os/os_detect.ts

```diff
@@ -7,8 +7,24 @@ declare global {
             msDetachStream: () => void;
         };
     }
+    interface Navigator {
```

Comment:
Type should be used instead of interface as it was agreed inside the team.

Reply:
We can't use type here. It's already declared globally. So, typescript will give me error if i make it type here.